src/babel/util.js

```diff
@@ -134,10 +174,18 @@ function _shouldIgnore(pattern, filename) {
   }
 }
 
+/**
+ * A visitor for Babel templates, replaces placeholder references.
+ */
+
 var templateVisitor = {
+
+  /**
+   * 360 NoScope PWNd
```

Comment:
How didn't I make this connection wtf.

![snoop DOGGY DOGG](https://media.giphy.com/media/CmjjuirIo0psk/giphy.gif)